rafthttp/stream.go

```diff
@@ -47,9 +45,9 @@ var (
 
 	// the key is in string format "major.minor.patch"
 	supportedStream = map[string][]streamType{
-		"2.0.0": {streamTypeMsgApp},
-		"2.1.0": {streamTypeMsgApp, streamTypeMsgAppV2, streamTypeMessage},
-		"2.2.0": {streamTypeMsgApp, streamTypeMsgAppV2, streamTypeMessage},
+		"2.0.0": {},
```

Comment:
I wants to remove this line and add new `"2.3.0"` line in another commit that bumps support map to 2.3 to make it clear.